packages/manager/apps/telecom-task/cypress/integration/standalone.spec.js

```diff
@@ -0,0 +1,46 @@
+import Page from '../support/po';
+
+const { SummaryPage } = Page.TelecomTaskPage;
+
+describe('Telecom Task should be standalone', () => {
+  beforeEach(() => {
+    cy.fixture('2api_pack_xdsl_tasks_details_all').as('fixture_2api_pack_xdsl_tasks_details_all');
+
+    cy.server();
+    cy.route(
+      'GET',
+      '/engine/apiv6/me',
+      'fixtures:me',
+    ).as('apiMe');
+
+    cy.get('@fixture_2api_pack_xdsl_tasks_details_all')
+      .then((json) => {
+        cy.route({
+          method: 'GET',
+          url: '/engine/2api/pack/xdsl/tasks/detail/all?page=1&perPage=5&sort=pack+DESC&status=todo+doing+cancelled+error+problem',
+          status: 200,
+          response: json,
+          headers: {
+            'content-type': 'application/json',
+          },
+        })
+          .as('2apiPackXdslTasksDetailsAll');
+      });
+  });
+
+  it('should load summary', () => {
```

Comment:
You should check if the title change when we change state (due to `@ovh-ux/ng-uirouter-title`)

Reply:
Addressed in https://github.com/ovh-ux/manager/pull/86/files#diff-8153ad87abcc8ec8738b36538944e296R46